package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.73.0",
   "private": true,
   "engines": {
-    "node": ">=20.15",
```

Comment:
This is the monorepo `package.json`, and not the one for the npm package `n8n`.
The target users of this file are the contributors of this project.
Are you building the app from source, instead of using the releases?